.github/workflows/frontend-checks.yml

```
@@ -0,0 +1,37 @@
+name: Frontend Checks
+
+on:
+  pull_request:
+    branches: ['main', 'develop']
```

```suggestion
    branches: ['main']
```
we don't have a develop branch